Backported patch from upstream to enable use of typed_ast 1.4.0

https://github.com/python/mypy/pull/6937

diff --git a/setup.py b/setup.py
index 1b9eb60..5146144 100644
--- a/setup.py
+++ b/setup.py
@@ -174,7 +174,7 @@ setup(name='mypy',
                                         ]},
       classifiers=classifiers,
       cmdclass=cmdclass,
-      install_requires = ['typed-ast >= 1.3.1, < 1.4.0',
+      install_requires = ['typed_ast >= 1.4.0, < 1.5.0',
                           'mypy_extensions >= 0.4.0, < 0.5.0',
                           ],
       extras_require = {
